### ISSUE TITLE
Maintenance of python_visualization module

### ISSUE THREAD

**Issue by clarkenj** [school-brainhack/school-brainhack.github.io#184]
## Module to maintain
Website: https://school-brainhack.github.io/modules/python_visualization/
Code: https://github.com/school-brainhack/school-brainhack.github.io/tree/main/content/en/modules/python_visualization

## Work to be done
Please ensure: 
- [ ] links, resources and exercises reflect the state-of-the-art 
- [ ] videos are of good quality and reflect the materials.  <br> 

If any update / re-write of the module is needed, feel free to suggest alternative material in a comment!
If you think major changes are needed, we suggest you open an issue for new modules.

**Comment by amandalin047** [comment]
I'd be happy to take this one!

**Comment by clarkenj** [comment]
Hi @amandalin047 , how are you getting on with maintenance of this module? Do you need any help?

**Comment by amandalin047** [comment]
Yes yes, I'm also looking at some of the Data Visualization exercise submissions I've received in our Taiwan hub. Will report the updates by the 21st!

**Comment by htwangtw** [comment]
@amandalin047 any updates? If all is working we can mark this as resolved!

**Comment by amandalin047** [comment]
Hey @htwangtw sorry for being a little bit late! I've checked the video, the exercise instructions, as well as some of the local Taiwanese students' exercise submissions. I think everything's pretty clear! Thanks :)
